lib/reaction-wheel/reaction-wheel.h

```diff
@@ -0,0 +1,24 @@
+#ifndef REACTIONwHEEL_H
```

Comment:
Consider moving this file to the include directory. That is where most of our functions are.

Reply:
it is separate for modularity reasons as the program gets larger

Reply:
Since we are working together why not? From the initial design spec user functions are in the include directory

Reply:
this thread seems to be same as the one above